q2_feature_table/_conditional_filter.py

```diff
@@ -0,0 +1,33 @@
+# ----------------------------------------------------------------------------
+# Copyright (c) 2016-2020, QIIME 2 development team.
+#
+# Distributed under the terms of the Modified BSD License.
+#
+# The full license is in the file LICENSE, distributed with this software.
+# ----------------------------------------------------------------------------
+
+
+import biom
+
+
+def filter_features_conditionally(table: biom.Table,
+                                  prevelance: float, abundance: float
+                                  ) -> biom.Table:
+    """
+    A function to perform joint filtering because it makes life better
+    """
+    num_observations, num_samples = table.shape
+    prevelance = prevelance * num_samples
+
+    # Calculates the filteering parameters on the original table
+    def _filter_f(v, id_, md):
+        return (v > abundance).sum() >= prevelance
+
+    # Normalized the table to get the prevelance
+    table_n = table.copy().norm(axis='sample', inplace=False)
+    table_s = table_n.filter(_filter_f, axis='observation', inplace=True)
```

Comment:
Since you _are_ modifying this table in place, assigning to a new variable isn't necessary.  Is there a reason you prefer assigning to a new variable?

I would also suggest using more descriptive variable names, e.g., `table_normalized`.